### PR TITLE
Add multiple domains

### PR DESCRIPTION
**Pull request by rand0mgh0st**
You can read the commit messages for descriptions of these sites.

### CODE REVIEW

**Comment by IceCodeNew** [comment]
@rootmelo92118 could you have a first look at this PR? I would like to review this PR based on your opinion.

**Comment by rootmelo92118** [comment]
> @rootmelo92118 could you have a first look at this PR? I would like to review this PR based on your opinion.

Basically, it can be merged directly, but I am not sure whether the domain `organicmaps.app` should be included in `category-orgs` or not. The reason is it is a map project, but I still cannot be sure whether it is an organization or just a project.

File: data/category-orgs [hunk 1]
```diff
@@ -16,6 +16,7 @@ f-droid.org
 globalsecurity.org
 ietf.org
 itu.int
+organicmaps.app
```
Comment (rootmelo92118) [inline data/category-orgs:19]: Here

**Comment by rand0mgh0st** [comment]
> I am not sure whether the domain organicmaps.app should be included in category-orgs or not. The reason is it is a map project, but I still cannot be sure whether it is an organization or just a project.

I'm also wondering where I should put `organicmaps.app`. Maybe we can put it directly into `geolocation-!cn`?

**Comment by rootmelo92118** [comment]
> > I am not sure whether the domain organicmaps.app should be included in category-orgs or not. The reason is it is a map project, but I still cannot be sure whether it is an organization or just a project.
> 
> I'm also wondering where I should put `organicmaps.app`. Maybe we can put it directly into `geolocation-!cn`?

Sounds like a good idea.

**Review by rootmelo92118 — APPROVED** [review]

**Comment by rand0mgh0st** [comment]
Sorry for bothering. Just found their (Organic Maps) another domain: `omaps.app`. I will add it soon.

**Review by IceCodeNew — APPROVED** [review]

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**